app/code/Magento/Persistent/Helper/Data.php

```diff
@@ -49,18 +49,26 @@ class Data extends \Magento\Framework\App\Helper\AbstractHelper
      */
     protected $_modulesReader;
 
+    /**
+     * @var \Magento\Store\Model\StoreManagerInterface
+     */
+    protected $storeManager;
+
     /**
      * @param \Magento\Framework\App\Helper\Context $context
      * @param \Magento\Framework\Module\Dir\Reader $modulesReader
      * @param \Magento\Framework\Escaper $escaper
+     * @param \Magento\Store\Model\StoreManagerInterface $storeManager
      */
     public function __construct(
         \Magento\Framework\App\Helper\Context $context,
         \Magento\Framework\Module\Dir\Reader $modulesReader,
-        \Magento\Framework\Escaper $escaper
+	\Magento\Framework\Escaper $escaper,
+	\Magento\Store\Model\StoreManagerInterface $storeManager
```

Comment:
Please fix code style of your changes (use 4 spaces instead of tab symbol)
Also, I think we need to use new parameter as optional

```suggestion
        \Magento\Framework\Escaper $escaper,
        \Magento\Store\Model\StoreManagerInterface $storeManager = null
```